person.rb

```diff
@@ -25,18 +30,13 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
+
```

Comment:
- [ ] Please make sure that the add rental takes date and book as parameters and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` as you see below :point_down: 
  
    ```ruby
    def add_rental(book, date)
        Rental.new(date, book, self)
    end
    ```